src/app/home/shared/forms/budgets-form/budgets-form.component.html

```diff
@@ -39,27 +39,27 @@
 
     <h4 class="category-header">Category</h4>
     <div class="categories-container">
-
-      <div *ngFor="let category of categories">
+      <div *ngFor="let category of categories; let i = index" class="category-container">
         <div class="category-element" (click)="selectCategory(category)">
           <i class="fas fa-circle category-circle"
              [class]="category.isSelected ?'selected-category-color' : 'standard-category-color'"></i>
           <i class="{{category.icon}} category-icon"></i>
         </div>
-        <p class="category-text">{{category.name}}</p>
-      </div>
-
-      <div class="category-add-element">
-        <i class="fas fa-circle category-circle standard-category-color"></i>
-        <i class="fas fa-plus category-icon"></i>
+        <p *ngIf="i < customCategoryIndex" class="category-text">{{category.name}}</p>
+        <text-input *ngIf="i === customCategoryIndex"
+                    [(ngModel)]="category.name"
+                    [ngModelOptions]="{standalone: true}"
+                    [isLabelVisible]="false"
+                    [isCheckIconVisible]="false"
+                    class="custom-category"></text-input>
       </div>
     </div>
 
     <input type="checkbox" id="cycle-checkbox" name="cycle-checkbox" value="true" [checked]="repeatCycle"
            (click)="isRepeatCycleFieldSelect()">
     <label for="cycle-checkbox"><span class="span-text">Repeat cycle</span></label>
 
-    <div class="btn-container" [class]="insideBudgetCard ? 'additional-margin' : null" >
+    <div class="btn-container" [class]="insideBudgetCard ? 'additional-margin' : null">
```

Comment:
`[class.additional-margin]="insideBudgetCard"`  or `[class="insideBudgetCard && 'additional-margin'"` should be enough here. No need for ternary.